const i16 NEGATIVE_I16 = -10
const double NEGATIVE_DOUBLE = -123.456

const i16 I16_CONST = 10
const i32 I32_CONST = 100000
const double DOUBLE_CONST = 123.456

const string DOUBLE_QUOTED_CONST = "hello"
const string SINGLE_QUOTED_CONST = 'hello'

const string CONST_WITH_SEP1 = "hello",
const string CONST_WITH_SEP2 = "hello";

const list<i32> I32_LIST_CONST = [1, 2, 3]
const list<double> DOUBLE_LIST_CONST = [1.1, 2.2, 3.3]
const list<string> STRING_LIST_CONST = ["hello", "world"]
const list<list<i32>> I32_LIST_LIST_CONST = [[1, 2, 3], [4, 5, 6]]
const list<list<double>> DOUBLE_LIST_LIST_CONST = [[1.1, 2.2, 3.3], [4.4, 5.5, 6.6]]
const list<list<string>> STRING_LIST_LIST_CONST = [["hello", "world"], ["foo", "bar"]]
